pom.xml

```diff
@@ -89,30 +89,11 @@
         <version>1.127.2</version>
         <scope>test</scope>
       </dependency>
-      <dependency>
-        <groupId>org.easymock</groupId>
-        <artifactId>easymock</artifactId>
-        <version>4.3</version>
-        <scope>test</scope>
-      </dependency>
       <dependency>
         <groupId>org.mockito</groupId>
         <artifactId>mockito-core</artifactId>
         <version>4.11.0</version>
         <scope>test</scope>
-        <exclusions>
```

Comment:
let objenesis come in for mockito, since we don't depend on it directly and are no longer needing to disambiguate it between easymock.